parachain/runtime/kintsugi/Cargo.toml

```diff
@@ -92,7 +92,8 @@ supply = { path = "../../../crates/supply", default-features = false }
 
 primitives = { package = "interbtc-primitives", path = "../../../primitives", default-features = false }
 
-parachain-info = { path = "../../../crates/parachain-info", default-features = false }
+parachain-info = { git = "https://github.com/paritytech/cumulus", branch = "polkadot-v0.9.16", default-features = false }
```

Comment:
this is the main reason why this pr is in draft. @gregdhill why do we use a fork of this crate? And would it be a breaking change to replace the one by the other?

Reply:
I can't recall exactly why we forked this, but as long as this works it should be fine. Can you check the storage names are the same?

Reply:
Storage is the same - the only difference I see is a different value for the default parachain id, so I think we're good